packages/core/test/interpreter.test.ts

```diff
@@ -639,10 +636,10 @@ describe('interpreter', () => {
       });
       const bState = toggleMachine.transition(activeState, { type: 'SWITCH' });
 
-      interpret(toggleMachine).start(bState);
+      interpret(toggleMachine, { state: bState }).start();
 
       setTimeout(() => {
-        expect(activityActive).toBeFalsy();
+        expect(activityActive).toBeTruthy();
```

Comment:
this change is suspicious - what's the reason behind it? 🤔

Reply:
As discussed, actions should not restart by default (still need to do that) but state invocations should be restarted.

Reply:
hm, quite interesting that it didn't work like that before - when it comes to invocations